lms/djangoapps/courseware/tabs.py

```diff
@@ -319,6 +319,7 @@ class DatesTab(EnrolledTab):
     type = "dates"
     title = ugettext_noop(
         "Dates")  # We don't have the user in this context, so we don't want to translate it at this level.
+    priority = 50
```

Comment:
I know 50 looks weird, but I simply kept the pattern of adding 10 (there currently exists priorities 10, 20, 30, 40)